macros/external/create_external_table.sql

```diff
@@ -93,6 +93,37 @@
     file_format = {{external.file_format}}
 {% endmacro %}
 
+{% macro sqlserver__create_external_table(source_node) %}
+
+    {%- set columns = source_node.columns.values() -%}
+    {%- set external = source_node.external -%}
+    {%- set partitions = external.partitions -%}
+
+    {# TODO add params for modifiying these options? #}
+    SET ANSI_NULLS ON;
+    SET QUOTED_IDENTIFIER ON;
```

Comment:
How common is it to change these values? I think it makes sense to check `external.ansi_nulls` and `external.quoted_identifier`, or possibly an `external.options` dict (which is comparable to other databases).

Reply:
great point. will do